packages/sdk/src/components/filter/condition/BaseFieldValue.tsx

```diff
@@ -0,0 +1,148 @@
+import type { IDateFilter, IFilterItem } from '@teable/core';
+import { CellValueType, FieldType } from '@teable/core';
+
+import { Input } from '@teable/ui-lib';
+import { useMemo } from 'react';
+
+import type { DateField, IFieldInstance } from '../../../model';
+import { NumberEditor, RatingEditor } from '../../editor';
+import {
+  FileTypeSelect,
+  FilterCheckbox,
+  FilterDatePicker,
+  FilterInput,
+  FilterLink,
+  FilterMultipleSelect,
+  FilterSingleSelect,
+  FilterUserSelect,
+} from '../component';
+import { EMPTY_OPERATORS, MULTIPLE_OPERATORS } from '../constant';
+import type { IFilterComponents } from '../types';
+
+interface IBaseFieldValue {
+  value: unknown;
+  operator: IFilterItem['operator'];
+  onSelect: (value: IFilterItem['value']) => void;
+  field?: IFieldInstance;
+  components?: IFilterComponents;
+}
+
+export function BaseFieldValue(props: IBaseFieldValue) {
+  const { onSelect, components, field, operator, value } = props;
+
+  const emptyComponent = <Input className="m-1 h-8 w-40 placeholder:text-[13px]" disabled />;
+  const showEmptyComponent = useMemo(() => {
+    const showEmpty = EMPTY_OPERATORS.includes(operator);
+    showEmpty && onSelect?.(null);
+    return showEmpty;
+  }, [operator, onSelect]);
+
+  if (showEmptyComponent) {
+    return emptyComponent;
+  }
+
+  const InputComponent = (
+    <FilterInput
+      placeholder="Enter a value"
+      value={value as string}
+      onChange={onSelect}
+      className="w-40"
+    />
+  );
+
+  switch (field?.type) {
```

Comment:
What about isMultipleCellvalue?

Reply:
later to support, that's early omissions